bindings_node/src/conversation.rs

```diff
@@ -231,7 +230,7 @@ impl Conversation {
         installation_ids: member
           .installation_ids
           .into_iter()
-          .map(|id| ed25519_public_key_to_address(id.as_slice()))
```

Comment:
Double-checking, the ID here is bytes and not string right?

Reply:
yep, should be